src/plugins/plan/inspector/components/PlanActivitiesView.vue

```diff
@@ -129,7 +129,7 @@ export default {
       this.selectedActivities = [];
       selection.forEach((selectionItem) => {
         if (selectionItem[0].context.type === 'activity') {
-          const activity = selectionItem[0].context.activity;
+          const activity = { ...selectionItem[0].context.activity };
```

Comment:
it doesn't seem like any of this was reactive? which part was proxied ?

Reply:
The the selection. Either way this fixes the problem.